src/Core/src/Platform/Windows/MauiWinUIApplication.cs

```diff
@@ -37,35 +27,29 @@ protected override void OnLaunched(UI.Xaml.LaunchActivatedEventArgs args)
 
 			Application = Services.GetRequiredService<IApplication>();
 
-			var mauiContext = new MauiContext(Services, MainWindow);
+			var winuiWndow = CreateNativeWindow(args);
 
-			var activationState = new ActivationState(mauiContext, args);
-			var window = Application.CreateWindow(activationState);
+			MainWindow = winuiWndow;
 
-			var content = window.View;
+			MainWindow.Activate();
 
-			var root = CreateRootContainer();
+			Services.InvokeLifecycleEvents<WindowsLifecycle.OnLaunched>(del => del(this, args));
+		}
 
-			var nativeContent = content.ToNative(mauiContext);
+		UI.Xaml.Window CreateNativeWindow(UI.Xaml.LaunchActivatedEventArgs? args = null)
+		{
+			var winuiWndow = new MauiWinUIWindow();
```

Comment:
Should we keep something virtual so users can still return a custom type here?

Reply:
I will do that in a separate PR as it means updates to the way windows are created. iOS/Windows can be virtual methods, but Android is going to need something with the bundles.